src/Connection.ts

```diff
@@ -275,22 +285,36 @@ const DEFAULT_MAX_RETRIES = 10
  */
 
 export default class Connection extends EventEmitter {
+
+    _debug: Todo
+    options: Todo
+    retryCount: Todo
+    wantsState: Todo
+    connectionHandles: Todo
+    step: Todo
+    socket?: Todo
+    didDisableAutoConnect?: Todo
+    isWaiting?: Todo
+    _isReconnecting: Todo
+    _backoffTimeout: Todo
+    sendID: Todo
```

Comment:
Yes, `onReconnecting` and `onMessage` work also ok without type definition. Marked the fields as `Todo` so that we'll define the actual type in the future (and the full benefit of the type system).